setup.py

```diff
@@ -8,6 +8,8 @@
 extras_require = {
     "aws": ["aiobotocore>=0.10.2"],
     "azure": ["azureml-sdk>=1.0.83"],
+    "digitalocean": ["python-digitalocean"],
+    "googlecloud": ["google-api-python-client"],
```

Comment:
This doesn't match the `ImportError` which says `pip install "dask-cloudprovider[gcp]"`.